pkg/sdk/tasks_gen.go

```diff
@@ -14,6 +14,7 @@ type Tasks interface {
 	ShowByID(ctx context.Context, id SchemaObjectIdentifier) (*Task, error)
 	Describe(ctx context.Context, id SchemaObjectIdentifier) (*Task, error)
 	Execute(ctx context.Context, request *ExecuteTaskRequest) error
+	TemporarilySuspendRootTasks(ctx context.Context, depId SchemaObjectIdentifier, id SchemaObjectIdentifier) (func() error, error)
```

Comment:
I think that a more clear API would be to have:
- method SuspendRootTasks, with taskId instead of depId, and returning the IDs of suspended tasks
- method ResumeTasks, which accepts the list of suspended tasks

Afterall, the behavior will be almost the same, but more explicit and readable for the user.

Reply:
I meant more that depId is a enigmatic name ;) I am okay with having two ids, but let's name them better.